website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
 
 ## May 2024
 
+#### dbt Cloud Launch Showcase event
+
+The following features are new or enhanced as part of our [dbt Cloud Launch Showcase](https://www.getdbt.com/resources/webinars/dbt-cloud-launch-showcase) event on May 14th, 2024:
+
+- **New:** [Unit tests](/docs/build/unit-tests) are now Generally Available (GA) in dbt Cloud. Unit tests enable you to test your SQL model logic against a set of static inputs.
+
+- <expandable alt_header="New: Preview of native support for Azure Synapse Analytics" > 
+
+  Native support in dbt Cloud for Azure Synapse Analytics is now available in [Public Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud)! 
+
+  To learn more, refer to [Connect Azure Synapse Analytics](/docs/cloud/connect-data-platform/connect-azure-synapse-analytics) and [Microsoft Azure Synapse DWH configurations](/reference/resource-configs/azuresynapse-configs).
+
+  Also, check out the [Quickstart for dbt Cloud and Azure Synapse Analytics](/guides/azure-synapse-analytics?step=1). The guide walks you through: 
+
+  - Loading the Jaffle Shop sample data (provided by dbt Labs) into Azure Synapse Analytics. 
+  - Connecting dbt Cloud to Azure Synapse Analytics.
+  - Turning a sample query into a model in your dbt project. A model in dbt is a SELECT statement.
+  - Adding tests to your models.
+  - Documenting your models.
+  - Scheduling a job to run.
+
+  </expandable>
+
+- **New:** [dbt Assist](/docs/cloud/dbt-assist) is a powerful AI feature, available in beta, helping you generate documentation, metrics, semantic models, and tests, saving you time and helping you deliver data that works. Available for dbt Cloud Enterprise accounts in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).
+
+- **New:** The VS Code extension [Power user for dbt Core and dbt Cloud](https://docs.myaltimate.com/arch/beta/) is now available in beta for [dbt Cloud CLI](/docs/cloud/cloud-cli-installation#vs-code-extension) users. The extension accelerates dbt and SQL development and includes features such as generating models from your source definitions or SQL, updating dbt model using natural language, and [more](https://docs.myaltimate.com/)!
+
+- **New:** MetricFlow enables you to now add [metrics as dimensions](/docs/build/ref-metrics-in-filters) to your metric filters to create more complex metrics and gain more insights. Available for all users.
+
+- **New:** Oauth login support via [Databricks](/docs/cloud/manage-access/set-up-databricks-oauth) is now GA to Enterprise customers.
+
+- <expandable alt_header="New: GA of dbt Explorer" > 
```

Comment:
@mirnawong1 - marketing advises us to note that Explorer's current features are GA, not the overall product, in case we launch new tiers in the future

"GA of dbt Explorer's features"
"dbt Explorer's current capabilities are now Generally Available for dbt Cloud customers. With Explorer, you can more easily navigate your dbt Cloud project – including models, sources, and their columns – to gain a better understanding  of its latest production or staging state."

Reply:
thanks @cafzal ! will fold those changes (cc @nghi-ly when you're back)